test/src/utils/get_utils_test.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:get/get.dart';
+
+// Identity util to convert to iterables
+dynamic _id(dynamic e) => e;
+
+void main() {
+  group('GetUtils.isNullOrBlank', () {
```

Comment:
so... about these tests

What do you thing of putting them on dynamic extensions?
I've done tests for all methods that are used as extension for String, num, and dynamic. It covered some of your tests, but it did not covered all of them.

So it would be awesome if you merge them
There are a few tests of isNullOrBlank there, what you think of put them there?

Reply:
okay, gonna merge them with those on dynamic exts

Reply:
@Nipodemos can you resolve this conversation once you've confirmed my test merges with those on dynamic are ok?